pkg/authorization/conditions/ips.go

```diff
@@ -0,0 +1,59 @@
+package conditions
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	ipapi "github.com/oauth2-proxy/oauth2-proxy/pkg/apis/ip"
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/sessions"
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/ip"
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/logger"
+)
+
+type ips struct {
+	ips      *ip.NetSet
```

Comment:
`ips.ips` seems like a weird name, what about this?
```suggestion
	netSet *ip.NetSet
```